### ISSUE TITLE
Issues with fetching all of my mirrored videos

### ISSUE THREAD

**Issue by AnushK-Fro** [kyleratti/a-centralized-mirror#21]
Hello,

I'm currently wanting to go through a bunch of my old links on the PublicFreakout subreddit and verify that they still exist & change some of the URL's :)

Here is the code I am using:
```
import requests
main = requests.get("https://api.amirror.link/mirroredvideos/all", headers = {'Content-Type': 'application/json', 'X-ACM-API-Token': '', 'X-ACM-Bot-Token': ''})
print(main.content)
```

I currently get a 422 error - {"status":{"status":422,"message":{}}}
I think it might be because of the amount of videos that I have mirrored - or it could be just me doing a dumb mistake 

Thanks!

**Comment by kyleratti** [comment]
Hey friend,

I'll look into this later today for you!

**Comment by kyleratti** [comment]
One quick thought: because this is a get request, does it work if you remove the content type header?

**Comment by kyleratti** [comment]
Okay, I think I figured it out. When I added the `GET /mirroredvideos/all` endpoint, I re-used some request checking logic that validated the inbound body. As such, you'll have to send a `GET` request that includes the following:

Headers:
```
Content-Type: application/json
X-ACM-Api-Token: xyz
X-ACM-Bot-Token: xyz
```

Body:
```json
{
	"data": {
		"redditPostId": "decscf",
		"url": "https://youtube.com/avideo"
	}
}
```

You can use that body exactly as-is; none of the content is actually used for anything, those properties just need to be there to pass validation.

This is a horrible violation of the HTTP spec and should be fixed, but see if you can get this working as described. The new version, in development and eventually going to be released (progress on #18), won't have this problem!

Let me know how it works out.

**Comment by AnushK-Fro** [comment]
Hello,

Thanks for the quick response! Yes, setting the parameters did indeed fix the issue 👍 
Final code for future reference:
```
from requests import get
from json import dumps

main = get(
    "https://api.amirror.link/mirroredvideos/all", 
    headers = {
        'Content-Type': 'application/json', 
        'X-ACM-API-Token': 'X', 
        'X-ACM-Bot-Token': 'X'
    }, 
    data = dumps({
        'data': {
            'redditPostId': 'decscf',
            'url': 'https://youtube.com/avideo'
        }
    })
)

print(main.content)
```

Thanks!
